cross/libaom/Makefile

```diff
@@ -51,3 +51,5 @@ endif
 ifeq ($(findstring $(ARCH),$(i686_ARCHS)),$(ARCH))
 CMAKE_ARGS += -DAOM_TARGET_CPU=x86
 endif
+
+ENV += CXXFLAGS=-D_GLIBCXX_USE_C99_MATH
```

Comment:
That's a hell of a good catch!
Does this impact DSM<7.0 ?  Or should a DSM version test be done?
Or even better:
```
ifeq ($(TC_GCC),7.3.0)
ENV += CXXFLAGS=-D_GLIBCXX_USE_C99_MATH
endif
```

Reply:
I don't see why it should be impacted negatively.

```
gcc 4.9.3 in DSM 6
The default, if no C++ language dialect options are given, is -std=gnu++98. 
https://gcc.gnu.org/onlinedocs/gcc-4.9.0/gcc/Standards.html

gcc 7.3 in DSM 7
The default, if no C++ language dialect options are given, is -std=gnu++14. 
https://gcc.gnu.org/onlinedocs/gcc-7.3.0/gcc/Standards.html
```

Seems to work fine on DSM 6. But I haven't done any C stuff in ages.